### PR TITLE
Add favorits  on the sidebar and show all notes that are currently ma…

### PR DESCRIPTION
**Pull request by jo541**
…rked as favorits.

### CODE REVIEW

**Comment by jo541** [comment]
This request is link to this issue https://github.com/levlaz/braindump/issues/121

File: app/main/views.py [hunk 1]
```diff
@@ -213,6 +213,23 @@ def notebooks():
         notebooks=notebooks,
         form=form)
 
+@main.route('/favorites', methods=['GET', 'POST'])
```
Comment (levlaz) [inline app/main/views.py]: This should only handle GET, there is nothing to POST here.

File: app/main/views.py [hunk 2]
```diff
@@ -213,6 +213,23 @@ def notebooks():
...
+@login_required
+def favorites():
+    if current_user.is_authenticated():
```
Comment (levlaz) [inline app/main/views.py]: This is not required since we have the @login_required decorator

File: app/main/views.py [hunk 3]
```diff
@@ -213,6 +213,23 @@ def notebooks():
...
+        notes = Note.query.filter_by(
+            author_id=current_user.id,
+            is_deleted=False,
+            is_favorite=True).order_by(
+            Note.updated_date.desc()).all()
+        return render_template('app/app.html', notes=notes)
+    else:
```
Comment (levlaz) [inline app/main/views.py]: See note above, this is automatically done if the user is not logged in.

Comment (jo541) [inline app/main/views.py]: My bad, It's okey now ^^.

2015-12-15 17:52 GMT+01:00 Lev Lazinskiy notifications@github.com:

**Comment by levlaz** [comment]
@jo541 This is great, thank you so much!

**Pull request merged**